Dockerfile

```diff
@@ -1,21 +1,48 @@
-FROM ubuntu:22.04 as tomcat
+FROM tomcat:9.0.85-jre11-temurin-jammy
```

Comment:
We could think about introducing build args like

```
ARG TOMCAT_VERSION=9.0.85
ARG JAVA_VERSION=11

FROM tomcat:${TOMCAT_VERSION}-jre{JAVA_VERSION}-temurin-jammy
```

This would give the build more flexibility.

We were previously using the `jdk` instead of `jre`. I don't know if there was a reason for this? Maybe @jodygarnett knows?

Reply:
I do not know, GeoServer is supposed to only require JRE.

Reply:
Agreed. But it might be tricky to get the JRE going, as it often requires less dependencies, which end up tripping either the startup, the map rendering, or specifically, the point rendering (fondconfig/harfbuzz not being installed or not completely working). Try it out with "topp:states", if that works, it _should_ be fine.

Reply:
The idea behind using the `JRE` image was that this would result in a smaller GeoServer image. I didn't realize that this could lead to side-effects in rendering for example. Thanks @aaime for this hint!

In 2de74a9bd1fff67b8c5d2a030938b7255b4536b1 I switched back to `JDK` in order to be consistent with the previous version and to avoid the difficulties of using the `JRE` (see above comment). Furthermore, due to the changes in https://github.com/geoserver/docker/pull/39, the `catalina.jar` has to be packed/unpacked using `jar` to exchange the version info which is not available in the `JRE` image.